schema.sql

```diff
@@ -0,0 +1,6 @@
+drop table if exists entries;
+create table entries (
+  id integer primary key autoincrement,
+  title text not null,
+  'text' text not null
+);
```

Comment:
Stage a few entries so that app does not appear empty out of the gate. Although in retrospect it's kind of silly, but maybe a hello and welcome message.